tests/iptables_test.rs

```diff
@@ -41,5 +44,52 @@ fn filter(ipt: iptables::IPTables, name: &str) {
     assert_eq!(ipt.exists("filter", name, "-j ACCEPT").unwrap(), false);
     assert_eq!(ipt.delete("filter", name, "-j DROP").unwrap(), true);
     assert_eq!(ipt.list("filter", name).unwrap().len(), 1);
+    assert!(ipt.execute("filter", &format!("-A {} -j ACCEPT", name)).is_ok());
+    assert_eq!(ipt.exists("filter", name, "-j ACCEPT").unwrap(), true);
+    assert_eq!(ipt.flush_chain("filter", name).unwrap(), true);
     assert_eq!(ipt.delete_chain("filter", name).unwrap(), true);
 }
+
+#[test]
+fn test_get_policy() {
+    let ipt = iptables::new(false).unwrap();
+
+    // filter
+    assert!(ipt.get_policy("filter", "INPUT").is_ok());
+    assert!(ipt.get_policy("filter", "FORWARD").is_ok());
+    assert!(ipt.get_policy("filter", "OUTPUT").is_ok());
+    // mangle
+    assert!(ipt.get_policy("mangle", "PREROUTING").is_ok());
+    assert!(ipt.get_policy("mangle", "OUTPUT").is_ok());
+    assert!(ipt.get_policy("mangle", "INPUT").is_ok());
+    assert!(ipt.get_policy("mangle", "FORWARD").is_ok());
+    assert!(ipt.get_policy("mangle", "POSTROUTING").is_ok());
+    // nat
+    assert!(ipt.get_policy("nat", "PREROUTING").is_ok());
+    assert!(ipt.get_policy("nat", "POSTROUTING").is_ok());
+    assert!(ipt.get_policy("nat", "OUTPUT").is_ok());
+    // raw
+    assert!(ipt.get_policy("raw", "PREROUTING").is_ok());
+    assert!(ipt.get_policy("raw", "OUTPUT").is_ok());
+    // security
+    assert!(ipt.get_policy("security", "INPUT").is_ok());
+    assert!(ipt.get_policy("security", "OUTPUT").is_ok());
+    assert!(ipt.get_policy("security", "FORWARD").is_ok());
+
+    // Wrong table
+    assert!(ipt.get_policy("not_existant", "_").is_err());
+    // Wrong chain
+    assert!(ipt.get_policy("filter", "_").is_err());
+}
+
+#[test]
+fn test_set_policy() {
```

Comment:
thank you for the change, it is more reliable now. However, I think it is better to use [ignore attribute](https://doc.rust-lang.org/book/testing.html#the-ignore-attribute) for this test to be optional.